README.md

```diff
@@ -1,103 +1,55 @@
 # The Charmed Operator Framework
 
-This Charmed Operator Framework simplifies [operator](https://charmhub.io/about) development
-for [model-driven application management](https://juju.is/model-driven-operations).
+The Charmed Operator Framework is a Python library (available as [ops](https://pypi.org/project/ops/))
+for developing and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
+to allow for clean, maintainable, and reusable code.
 
-Operators emerged from the Kubernetes community; an operator is software that drives lifecycle
-management, configuration, integration and daily actions for an application. Operators simplify
-software management and operations. They capture reusable app domain knowledge from experts in a
-software component that can be shared.
+Charms written with the Charmed Operator Framework support Kubernetes using
```

Comment:
I think majoring on "charms" is the right way to go, but I'd include this (short) definition/clarification in the README too:

```
A charm is an operator - business logic encapsulated in reusable software packages that automate every aspect of an application's life. 
```